BraveShared/de.lproj/BraveShared.strings

```diff
@@ -301,6 +301,9 @@
 /* DuckDuckGo callout title. */
 "DDGCalloutTitle" = "Private Suche über DuckDuckGo?";
 
+/* Name for default Brave theme. */
+"defaultThemeName" = "Brave-Standarddesign";
+
```

Comment:
This needed?

Reply:
Ah it looks like it's in every language. Nevermind

Reply:
this German translations looks funny :D it's for custom home page/themes setting

Reply:
Haha yeah the key made me think we accidentally snuck in a static file name dictionary key or something into the strings file 😆